store/docs/_index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)